TestMain.py

```diff
@@ -24,7 +27,7 @@ def tearDownClass(cls):
             shutil.rmtree('resources/specimin')
         except Exception as e:
             print(f"Error occurred {e}")
-        # removing any issue project cloned in resources
+        # removing any issue project cloned in resources   
```

Comment:
Please don't make unnecessary white-space changes like this. They increase review load for no reason.

You might want to consider using an automated formatting tool to avoid this.